spec-tests/README.md

```diff
@@ -1,11 +1,35 @@
 # Candlepin - Java base spec tests
 Java spec tests for candlepin. 
```

Comment:
Can we add a link from the top-level CONTRIBUTING.md (https://github.com/candlepin/candlepin/blob/master/CONTRIBUTING.md#spec-tests) to here? Specifically from the java spec-test section there. It's nice having extra README files, but if they're hidden/unknown to users they're not useful and tend to even get stale faster (I didn't even realize there was a README in the spec-tests dir until now)

Reply:
That's a really good point! I added a link in the CONTRIBUTING.md file.